### DISCUSSION TITLE
Successfully ran on Yamaha AVR rx v677

### DISCUSSION THREAD

**Discussion by arunprasaddoss** [coffeegreg/YTuner#27] [General]
Let me assure again that the Yamaha v677 is working when I am running yTuner as a service from my Ubuntu Server. 

Thanks a lot for this great work. , 😄 

Now working on creating my stations list.

**Comment by TheBossME** [comment]
Nice to hear, thank you for sharing this. 
Greg do a very good job with ytuner project !. 

Happy listening to worldwide free Internet radio and have a great sunny day full of wonderful music !

Beatrice
